lisa/target.py

```diff
@@ -659,6 +661,66 @@ def install_tools(self, tools):
             self.target.install(tool)
             self._installed_tools.add(tool)
 
+    def build_kmodule(self, kernel_src, kmodule_path):
+        """
+        Compile kernel module that we want to load at the target at some point
+
+        :param kernel_src: The path to the kernel source tree to use to build the module
+        :type kernel_src: str
+
+        :param kmodule_path: The path to the module to build
+        :type kmodule_path: str
+        """
+        self.get_logger().info('Building kernel module [%s] using [%s] kernel tree', kmodule_path, kernel_src)
+
+        cmd = os.path.join(LISA_HOME, 'tools/kmodules/clean_module')
+        failed = subprocess.call([cmd, kernel_src, kmodule_path])
```

Comment:
`subprocess.check_call` provides this check already, so this:
```
         failed = subprocess.call([cmd, kernel_src, kmodule_path]) 
        if failed:
            raise RuntimeError('Failed to clean kernel module: %s', kmodule_path) 
```
can be simplified into this:
```
         subprocess.check_call([cmd, kernel_src, kmodule_path]) 
```